layers/mpls_echo_test.go

```diff
@@ -0,0 +1,933 @@
+// Copyright 2018 GoPacket Authors. All rights reserved.
```

Comment:
Can you make the testfile more in line with the existing tests? (without the helper functions)